spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -72,4 +72,135 @@
       expect(page).to have_current_path(idv_hybrid_handoff_url)
     end
   end
+
+  describe 'navigating to How To Verify from Agreement page in 50/50 state' do
+    context 'opt in false at start but true during navigation' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
+
+      it 'should be on the hybrid handoff page' do
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+        page.refresh
+      end
+
+      it 'should be not be bounced back from hybrid handoff to how to verify' do
+        expect(page).not_to have_current_path(idv_how_to_verify_url)
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+    end
+
+    context 'opt in true at start but false during navigation' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
+
+      it 'should be on the how to verify page' do
+        expect(page).to have_current_path(idv_how_to_verify_url)
+      end
+
+      it 'should be on the hybrid handoff page when opt in is false' do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+        page.refresh
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+    end
+  end
+
+  describe 'navigating backwards from How to Verify page in 50/50 state' do
+    context 'Going back from Hybrid Handoff with opt in disabled midstream' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+        complete_how_to_verify_step
+      end
+
+      it 'should be on the how to verify page' do
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+
+      it 'should be not be bounced back to how to verify with opt in disabled midstream' do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+        page.go_back
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+    end
+
+    context 'Going back from Hybrid Handoff with opt in enabled midstream' do
```

Comment:
I don't see tests around going back from IPP.

Reply:
Since we aren't toggling IPP with this deploy, I wasn't sure that I wanted to capture it?

Reply:
This is a new flow/path so I thought a good idea, maybe too much? 

Previously, if the user was on verify/document_capture#prepare and went back- they'd go back to document capture- but in this case- they should go back to how to verify.

Reply:
Ok, sorry I misunderstood. Yeah let me write some specs handling the back from document capture cases. Thanks!